Tests/PublisherTests/DefaultPublisher/DefaultPublisherTests.swift

```diff
@@ -861,3 +861,12 @@ class DefaultPublisherTests: XCTestCase {
         waitForExpectations(timeout: 10.0)
     }
 }
+
+extension ErrorInformation {
```

Comment:
```suggestion
private extension ErrorInformation {
```

Reply:
Or, if you're using it in more than one test file, perhaps put it in a test helpers file?

Reply:
Also, `isEqual` is a misleading name I think, since it doesn't compare the `cause`. Perhaps a name that makes it clear what you want to check is the same about the two errors?

Reply:
> Also, `isEqual` is a misleading name I think, since it doesn't compare the `cause`. Perhaps a name that makes it clear what you want to check is the same about the two errors?

what's your proposal then?

Reply:
`isEqualIgnoringCause(to:)`?

Reply:
I've reverted this commit in case of https://github.com/ably/ably-asset-tracking-swift/runs/4534264133?check_suite_focus=true#step:4:3861

Reply:
Sorry, I don't understand why you reverted it? Isn't the solution 

> Or, if you're using it in more than one test file, perhaps put it in a test helpers file?

Reply:
Sorry for the confusion, I've moved it to separated file 25da7e28529756d238551dd2be3a44052317475d
About method naming, I want to leave it as it is `isEqual` is fine IMO.